docs/en/faq.md

```diff
@@ -306,7 +306,7 @@ No need to set up a node, just set up a wallet (version 2.0.1) or Qbundle (2.0)
 
 **53**)** What pool should I join?**
 
-First of all, read question 9, after you have understood that it depends on the size (and how patient you are) select from the following list: https://www.ecomine.earth/burstpools/
+First of all, read question 9, after you have understood that it depends on the size (and how patient you are) select from the following list:
```

Comment:
This question/answer doesnt seem to make sense. Looks like question 9 has changed. It also will need a new pools list.

Reply:
According to `git blame` this was changed to Q15

Reply:
Merged, but didn't add a new pools list, so there is no "following list" now...